.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
           ]
           args: ['--max-line-length=88', 
                  '--docstring-convention=numpy', 
-                 '--per-file-ignores=**/tests/**.py:D functional-tests/**:D', 
+                 '--per-file-ignores=**/tests/**.py:D functional-tests/**:D,E501,W291',
```

Comment:
Can we add a remark what each means or a link to where you got this codes from? 
When someone reads the code he/she starts looking it online and it takes time.

Reply:
Done